tests/excel/test_merged_cells_edge_cases.py

```
@@ -0,0 +1,445 @@
+        ws = wb.active
+
+        # 1000個の小さな結合セルを作成
+        for i in range(1000):
+            row = i + 1
+            ws[f"A{row}"] = f"データ{i}"
+            ws[f"B{row}"] = f"値{i}"
+            ws.merge_cells(f"A{row}:B{row}")
+
+        wb.save(file_path)
+
+        # メモリ使用量を監視しながら処理（psutil代替実装）
```

Ruff (0.11.9)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)